anaconda.spec.in

```diff
@@ -348,7 +348,7 @@ Requires: iputils
 
 %description dracut
 The 'anaconda' dracut module handles installer-specific boot tasks and
-options. This includes driver disks, kickstarts, and finding the anaconda
+options. This includes driver disks, kick-starts, and finding the anaconda
```

Comment:
This is not correct change.

Reply:
I have dropped spelling fixes from this PR to address them as a change in rpmlint config (disable).